docs/release-process.md

```diff
@@ -7,13 +7,13 @@ Once the release is on the CDN the auto-update process will start downloading th
 
 These instructions are for creating version 0.1.0-alpha.5, but the steps are the same for releases with no pre-release tag.
```

Comment:
We could also update the version `0.1.0-alpha.5` here to `0.1.0-alpha5`.

Reply:
Good catch! Fixed: https://github.com/Automattic/studio/pull/450/commits/cb6beee00ff48761caab6c13cd99be557e532ce3